config/initializers/content_security_policy.rb

```diff
@@ -10,9 +10,9 @@
     policy.font_src    :self, "https://fonts.gstatic.com"
     policy.img_src     :self, "https://secure.gaug.es", "https://gravatar.com", "https://secure.gravatar.com", "https://*.fastly-insights.com"
     policy.object_src  :none
-    policy.script_src  :self, "https://secure.gaug.es", "https://www.fastly-insights.com"
+    policy.script_src  :self, "https://secure.gaug.es", "https://www.fastly-insights.com", "http://secure.gaug.es/track.js"
```

Comment:
Please remove this change. I don't see any CSP violation for secure.gaug.es on rubygems.org.